src/Microsoft.DotNet.Build.Tasks/Targets/test-runtime/project.json

```diff
@@ -36,9 +37,9 @@
        "System.Threading.Thread": "4.0.0-beta-*",
        "System.Xml.ReaderWriter": "4.0.10",
        "System.Xml.XDocument": "4.0.10",
-       "xunit": "2.1.0-beta3-*",
```

Comment:
I don't believe it has been reported anywhere but I started hitting it locally after we switched to 2.1 version of xunit and it has been polluting my local projects.